run_selenium_system_test.sh

```diff
@@ -93,5 +94,5 @@ function join_by {
 # Doing `sbt -Doption1=value1 -Doption2="value2 with spaces" selenium:test` works on some
 # environments but doesn't work with others - here we run sbt and add java system properties
 # within the sbt session and then run the tests
-JAVA_OPTS=$(get_java_opts $1 $2 ${@:3})
+JAVA_OPTS=$(get_java_opts $1 $2 $3 ${@:4})
 sbt "; set javaOptions in SeleniumTest ++= Seq(${JAVA_OPTS}); selenium:test"
```

Comment:
Suggest a comment with a link telling people where to get the drivers and a note that mac users will have to rename the downloaded 'chromedriver' to 'chromedriver_mac'